apps/web/src/views/BuyCrypto/components/AccordianDropdown/AccordianItem.tsx

```diff
@@ -104,7 +105,7 @@ function AccordionItem({
 
   let finalQuote = quote.amount
   if (quote.provider === 'MoonPay') finalQuote = MoonapyAmt
-  if (quote.provider === 'BinanceConnect') finalQuote = MercuryAmt
+  if (quote.provider === 'Mercuryo') finalQuote = MercuryAmt
```

Comment:
Although the name won't change frequently, but the best practice would be using a enum to compare, so you don't need to update the code everywhere when the provider's name changed